src/elasticsearch/user-elasticsearch.service.ts

```
@@ -256,6 +258,8 @@ export class UserElasticsearchService implements OnModuleInit {
       gender: profile.gender,
       dob: profile.dob,
       country: profile.country,
+      permanentCountry: profile.permanentCountry ?? '',
```

ESLint

[error] Replace `''` with `""`

(prettier/prettier)